packages/block-library/src/gallery/index.php

```diff
@@ -104,7 +104,17 @@ function block_core_gallery_render( $attributes, $content ) {
 	// Set the CSS variable to the column value, and the `gap` property to the combined gap value.
 	$style = '.wp-block-gallery.' . $class . '{ --wp--style--unstable-gallery-gap: ' . $gap_column . '; gap: ' . $gap_value . '}';
 
-	gutenberg_enqueue_block_support_styles( $style, 11 );
+	// If we're on WordPress >= 6.1, we can use `wp_enqueue_block_support_styles`, as it supports
+	// `$priority` as its second argument. Otherwise, we have to fall back to using our
+	// `gutenberg_enqueue_block_support_styles` shim.
+	if (
+		function_exists( 'wp_enqueue_block_support_styles' ) &&
+		2 === count( ( new ReflectionFunction( 'wp_enqueue_block_support_styles' ) )->getParameters() )
+	) {
```

Comment:
An alternative would be

```suggestion
	if ( version_compare( get_bloginfo( 'version' ), '6.1', '>=' ) ) {
```

but I find that less semantic. It also is less accurate, e.g. during the 6.1 alpha phase (now): It would evaluate to `false` even once https://github.com/WordPress/wordpress-develop/pull/3158 is merged (and used in a local install).